.github/PULL_REQUEST_TEMPLATE/Improvement.md

```diff
@@ -12,6 +12,7 @@ about: You have some improvement to make badge-poser better? 🎁
 | New Feature | yes
 | RFC         | yes/no
 | BC Break    | yes/no
+| Issue       | Fix #...
```

Comment:
`Close`?

Reply:
I was unsure which to use

Reply:
it should be the same

Reply:
Yeah, I have seen both solutions but for me `Close` is more explicit.